### PR TITLE
{perf}[gompi/2020a] Scalasca v2.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

collapsing from `2020a` branch into `develop`, see ~~#10421~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 11 (10 easyconfigs in this PR)
node3119.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/3b7aca8a05942fdc33f860cc72f44560 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/2322bac478b684114bfe0f6ff1ec8421 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node3409.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/33986c2dda4a38fe60324c89c5056e47 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node372.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c80a2a4fe2891fdca4e02b649d92f61e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/8314c9e07946757bd48fab31723ba58c for a full test report.

File: easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.5-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+builddependencies = [
+    # use same binutils version that was used when building GCCcore
+    ('binutils', '2.34'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/opari2', 'include/opari2/pomp2_lib.h'],
+    'dirs': [],
+}
+
```
Comment (lexming) [inline easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.5-GCCcore-9.3.0.eb:43]:
```suggestion

sanity_check_commands = ['opari2-config --help']

```

File: easybuild/easyconfigs/o/OTF2/OTF2-2.2-GCCcore-9.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+]
+
+configopts = '--enable-shared'
+
+sanity_check_paths = {
+    'files': ['bin/otf2-config', 'include/otf2/otf2.h',
+              'lib/libotf2.a', 'lib/libotf2.%s' % SHLIB_EXT],
+    'dirs': [],
+}
+
```
Comment (lexming) [inline easybuild/easyconfigs/o/OTF2/OTF2-2.2-GCCcore-9.3.0.eb:47]:
```suggestion

sanity_check_commands = ['otf2-config --help']

```

File: easybuild/easyconfigs/s/Score-P/Score-P-6.0-gompi-2020a.eb [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+builddependencies = [
+    ('CubeLib', '4.4.4'),
+    ('CubeWriter', '4.4.3'),
+    # Unwinding/sampling support (optional):
+    ('libunwind', '1.3.1'),
+]
+
+dependencies = [
+    # binutils is implicitly available via GCC toolchain
+    # ('binutils', '2.34'),
```
Comment (lexming) [inline easybuild/easyconfigs/s/Score-P/Score-P-6.0-gompi-2020a.eb:39]: These build deps should be moved to runtime dependencies as discussed in #10397 and #10496

```suggestion
dependencies = [
    ('CubeLib', '4.4.4'),
    ('CubeWriter', '4.4.3'),
    # Unwinding/sampling support (optional):
    ('libunwind', '1.3.1'),
```

Comment (boegel) [reply easybuild/easyconfigs/s/Score-P/Score-P-6.0-gompi-2020a.eb]: fixed in 137f877

File: easybuild/easyconfigs/o/OTF2/OTF2-2.2-GCCcore-9.3.0.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/o/OTF2/OTF2-2.2-GCCcore-9.3.0.eb:50]: fixed in fe8b5dc

File: easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.5-GCCcore-9.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/o/OPARI2/OPARI2-2.0.5-GCCcore-9.3.0.eb:43]: fixed in fe8b5dc

File: easybuild/easyconfigs/s/Score-P/Score-P-6.0-gompi-2020a.eb (hunk 3 above)
Comment (boegel) [reply easybuild/easyconfigs/s/Score-P/Score-P-6.0-gompi-2020a.eb]: and also for OTF2's SIONlib dep in 91a3bff

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node3412.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/a04da75747820916916c91c854ecff36 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/05360d33087d6cb53446540e4382ece8 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node101.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/cdee2b462206e8c1a0bf1f8b01660749 for a full test report.

**Pull request merged**